# Extract the best locations that entirely cover a map
# author: Tim Patten, t.patten@acfr.usyd.edu.au

uint8 tree_depth = 14           # the tree depth for the search
float32 robot_height = 0.7      # the rheight of the onboard sensor (assumes this is the top of the robot)
float32 robot_outer_range = 2.0 # the outer range limit of the onboard kinect sensor
float32 robot_inner_range = 1.0 # the inner range limit of the onboard kinext sensor
float32 robot_radius = 0.5      # the radius of the robot
float32 step_size = 0.5         # the step size to extract the initial exhaustive set of locations before the optimization
int8 maximum_iterations = 20    # the maximum number of iterations allowed
bool visualize_on = 0           # visualize the locations

octomap_msgs/Octomap map  # the octomap

---

# list of locations (no specific order)
geometry_msgs/Point[] positions
